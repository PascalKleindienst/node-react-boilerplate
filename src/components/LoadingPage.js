// ==============================================
// LOADING PAGE COMPONENT =======================
// ==============================================
import React from 'react';

const LoadingPage = () => (
    <div className="loader">
        <img className="loader__image" src="/images/loader.svg" />
    </div>
);

export default LoadingPage;
