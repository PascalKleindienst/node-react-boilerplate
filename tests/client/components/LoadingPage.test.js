// ==============================================
// LOADING PAGE TEST ============================
// ==============================================
import React from 'react';
import { shallow } from 'enzyme';
import LoadingPage from '../../../src/components/LoadingPage';

describe('my LoadingPage component', () => {
    test('should render correctly', () => {
        const wrapper = shallow(<LoadingPage />);
        expect(wrapper).toMatchSnapshot();
    });
});