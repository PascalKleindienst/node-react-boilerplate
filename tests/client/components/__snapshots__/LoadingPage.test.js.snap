// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`my LoadingPage component should render correctly 1`] = `
<div
  className="loader"
>
  <img
    className="loader__image"
    src="/images/loader.svg"
  />
</div>
`;
